tests/author_spec.rb

```diff
@@ -0,0 +1,42 @@
+require_relative '../Game/author'
+require_relative '../Game/game'
+
+RSpec.describe Author do
+  let(:author) { Author.new('John', 'Doe') }
+
+  describe '#initialize' do
+    it 'generates a random ID between 1 and 1000' do
+      expect(author.id).to be_between(1, 1000)
+    end
+
+    it 'sets the first name and last name' do
+      expect(author.first_name).to eq('John')
+      expect(author.last_name).to eq('Doe')
+    end
+
+    it 'initializes an empty array of items' do
+      expect(author.items).to eq([])
+    end
+  end
+
+  describe '#add_item' do
+    let(:item) { double('Item') }
+
+    it 'adds the author to the item\'s list of authors' do
+      expect(item).to receive(:add_author).with(author)
+      author.add_item(item)
+    end
```

Comment:
- [ ] Kindly fix this failing test, all the tests should pass(green).

![tests](https://github.com/Newtayo/Ruby-Capstone/assets/53124548/aa5a6871-da6e-46d9-9d47-c59663564332)